governance/working groups/working-group-charter.md

```diff
@@ -0,0 +1,41 @@
+# Working group charter template
+
+## Overview
+
+This section should summarise in one or two paragraphs the Working group proposal, for any community member to read and quickly grasp the essence of the WG.
+
+## Background
+
+This section should dive into more detail on what the WG aims to address, why it is undertaking this work, and reference any background material/pre-existing work that this proposal builds on.
+
+## Target outputs
+This section should contain the target outputs of the WG
+
+## Meeting mechanisms
+This section should include details on how, where and when the WG intends to meet
+
+## Communication mechanisms
+This section should outline how the WG intends to communicate progress with its members, the UK TRE Community more widely, and any other interested stakeholders.
```

Comment:
We could think about including a section on 'Core stakeholders', which are named individuals or organisations that need to at a minimum make up WG members. Not sure if this is overkill rn though

Reply:
I wouldn't bother with defining core vs non-core, but it might be nice to ask people to list out notable stakeholder groups? This helps us assess whether they're aware of who their WG will potentially impact, and whether they understand the broad remit of UK TRE (i.e. not just NHS/SDEs).